### ISSUE TITLE
Creating arguments bundle w/o need to call build()

### ISSUE THREAD

**Issue by MarcinOrlowski** [sockeqwe/fragmentargs#58]
I got couple of Fragments that I usually handle with FragmentArgs' generated builders. Now I am in need to passing such fragment to another library which expects me to hand `class` and arguments bundle separately. I'd like to still use FragmentArgs' builder to create the bundle but the only option I found out now is:

    Foo frag = new FooBuilder(args).build();
    Bundle args = frag.getArguments();

I'd like to skip `build()` here, but builder's `mArguments` is private and there's no getter so I am unable to get it other way than thru fragment. But this is bad as I do not want the fragment instantiated now. Can you consider changing the code a bit so such use case is supported w/o instantiating the fragment?

**Comment by sockeqwe** [comment]
I think this should be possible. Adding a method `Builder.asBundle()` should do the job.

**Comment by MarcinOrlowski** [comment]
or perhaps adding `getArguments()` (for naming consistency)? So it would just return the bundle as it is now in the moment of getting.

**Comment by sockeqwe** [comment]
I would like to not return the original (internal) bundle, but rather a copy of the original bundle (**immutability**). Hence `asBundle()` or maybe `toBundle()` seems to me a more appropriated name, but I'm open to discuss that

**Comment by MarcinOrlowski** [comment]
`toBundle()` is fine, so is i.e. `buildBundle()` (as bonus - it would nicely show up in autocomplete :).

As you tagged it for `4.0`, more important for me now is what's ETA for 4.0?

**Comment by sockeqwe** [comment]
it's implemented, you can use latest SNAPSHOT 4.0.0 from

```
 repositories {
    maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
 }
```

**Comment by MarcinOrlowski** [comment]
Works fine. Thanks for quick implementation.
